.gitignore

```diff
@@ -14,7 +14,6 @@ yarn-error.log*
 
 # Editor directories and files
 .idea
-.vscode
```

Comment:
I am guessing sometimes some .vscode files might be generated when working in the IDE. I guess that is why we had this line. We could keep it and just unignore the extensions file? 

what about this:

```
.vscode
!.vscode/extensions.json
```

Reply:
ah you're right!